storybooknative/src/Button/index.js

```diff
@@ -132,6 +144,24 @@ const Button = (props: Props) => {
     width = props.width || "auto";
   }
 
+  if (Platform.OS === "android") {
```

Comment:
That's much better. 
However, are you sure that all the components need access to `...rest`?
Maybe to improve even further, this could be in a different file `index.android.js`, so that the bundle size for other platform is not bloated with non-necessary code. @tbergq What do you think?

Reply:
I think that the .android and .ios suffixes should be used with care and only when utmost needed. Like the datepicker component. 

I doubt the effect on the bundle size is that significant.